.gitignore

```diff
@@ -1,20 +1,27 @@
-/CMake*
-!/CMakeLists.txt
-/*.cmake
-/Makefile
-/gmock/
-/*.cpp
-/include/
-/ts/ts.qrc.depends
-/chewing-editor.desktop
-/chewing-editor
-/chewing-editor.1
-/chewing-editor.1.gz
-/chewing.json
-/*.a
-/run-test*
-/ts/*.qm
-/ui_*.h
-/Testing/
-/coverage*
-/chewing-editor.dSYM/
+CMake*
+!CMakeLists.txt
+*.cmake
```

Comment:
Oops, it breaks the observing for `cmake/*.cmake` files from git.
Maybe we should add the rule `!cmake/*.cmake` or other rule to prevent ignoring these cmake modules.